crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/service/RepOrderService.java

```diff
@@ -14,13 +14,20 @@
 @RequiredArgsConstructor
 public class RepOrderService {
 
+    // Crown Court Rep Decisions
     public static final String GRANTED_FAILED_MEANS_TEST = "Granted - Failed Means Test";
     public static final String FAILED_CF_S_FAILED_MEANS_TEST = "Failed - CfS Failed Means Test";
     public static final String GRANTED_PASSED_MEANS_TEST = "Granted - Passed Means Test";
     public static final String REFUSED_INELIGIBLE = "Refused - Ineligible";
     public static final String GRANTED_PASSPORTED = "Granted - Passported";
     public static final String FAILED_IO_J_APPEAL_FAILURE = "Failed - IoJ Appeal Failure";
 
+    // Crown Court Rep Types
+    public static final String CROWN_COURT_ONLY = "Crown Court Only";
```

Comment:
I still feel like these constants belong elsewhere, either in a dedicated class or enumeration.

Reply:
Done.